app/commands/thredded/notify_following_users.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+module Thredded
+  class NotifyFollowingUsers
+    def initialize(post)
+      @post = post
+    end
+
+    def run
+      return unless targetted_users.present?
```

Comment:
I've gotten into the habit of erring on the side of using `if` instead of `unless` where the possibility is there. In this case we can represent this line similarly with

```ruby
return if targeted_users.empty?
```

Also - ~~targetted~~ => targeted. See [here](http://www.future-perfect.co.uk/grammar-tip/is-it-targetted-or-targeted/)